test/56.js

```diff
@@ -19,5 +20,20 @@ describe('return smallest number of coins required to return the change',
   it('hard case [1,2,3,6,7], total=12', () => {
     expect(solution([1, 2, 3, 6, 7], 12)).to.equal(2);
   });
+  it('simplest case [1], total=1', () => {
+    expect(solution2([1], 1)).to.equal(1);
```

Comment:
Can you call your `solution2` right behind `solution` in line 9?  Refer to how it was done in this test file: https://github.com/llipio/algorithms/blob/master/test/13.js